agent/config/testdata/TestRuntimeConfig_Sanitize.golden

```diff
@@ -74,7 +74,8 @@
     "CAPath": "",
     "Cache": {
         "EntryFetchMaxBurst": 42,
-        "EntryFetchRate": 0.334
+        "EntryFetchRate": 0.334,
+        "Logger": null
```

Comment:
TODO: see if this json marshal still works with a non-nill Logger.

Reply:
I tried it out, and it serialized just fine. Prints `map[string]interface{}`, since the struct has no exported fields.